### PR TITLE
Histogram and csv support for httpTrace fields

### PR DESCRIPTION
**Pull request by JRonak**
pr for #7

### CODE REVIEW

File: requester/print.go [hunk 1]
```diff
@@ -119,31 +135,74 @@ func (r *report) print() {
...
+			fmt.Printf("  Fastest:\t%4.4f secs\n", fastest)
+			fmt.Printf("  Slowest:\t%4.4f secs\n", slowest)
+			printHistogram(r.res, fastest, slowest)
+		}
 	}
 }
 
 func (r *report) printCSV() {
+	if r.trace {
+		fmt.Printf("response-time,DNS+Dialup,DNS,request-write,respone-wait,respone-read\n")
```
Comment (rakyll) [inline requester/print.go]: why not?

```
fmt.Printf("response-time")
if r.trace {
    fmt.Printf(",DNS+Dialup,DNS,request-write,respone-wait,respone-read")
}
fmt.Println()
```

File: requester/print.go [hunk 2]
```diff
@@ -119,31 +135,74 @@ func (r *report) print() {
...
+	}
 	for i, val := range r.lats {
-		fmt.Printf("%v,%4.4f\n", i+1, val)
+		if r.trace {
```
Comment (rakyll) [inline requester/print.go]: Same here,

```
fmt.Printf("%v,%4.4f", r.lats[i])
if t.trace {
     fmt.Printf(",%4.4f,%4.4f,....
}
fmt.Println()
```

File: requester/print.go [hunk 3]
```diff
@@ -119,31 +135,74 @@ func (r *report) print() {
...
+			var slowest, fastest float64
+			sort.Float64s(r.dns)
+			sort.Float64s(r.conn)
+			sort.Float64s(r.delay)
+			sort.Float64s(r.res)
+			sort.Float64s(r.req)
+			fmt.Printf("\n\nDetailed Report:\n")
+
+			slowest, fastest = r.conn[len(r.conn)-1], r.conn[0]
+			fmt.Printf("\nDNS+dialup:\n")
```
Comment (rakyll) [inline requester/print.go]: Let's make this a func.

```
func printSection(name string, avg, fastest, slowest float64) {
 			fmt.Printf("\n%s:\n",name)
 			fmt.Printf("  Average:\t%4.4f secs\n", avg)
 			fmt.Printf("  Fastest:\t%4.4f secs\n", fastest)
 			fmt.Printf("  Slowest:\t%4.4f secs\n", slowest)
}
```

**Comment by JRonak** [comment]
Done

**Comment by JRonak** [comment]
@rakyll I think i have messed up this PR, shall I close it and come up with a clean one?
or any other suggestion?

**Comment by rakyll** [comment]
Sure, please open a new PR or anything you like.

**Pull request closed without merging**